packages/react/src/components/FormLabel/FormLabel.stories.mdx

```diff
@@ -0,0 +1,53 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import FormLabel from './FormLabel.tsx';
+
+export const meta = {
+  component: FormLabel,
+  title: StoryConfig.FormLabel.hierarchy,
+};
+
+<Meta title={meta.title} component={meta.component} />
+
+export const Template = args => <FormLabel {...args} />;
+
+# FormLabel
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+
+## Overview
+
+API reference docs for the React FormLabel component. Learn about the props, CSS, and other APIs of this exported module.
+
+For examples and details on the usage of this React component, visit the component demo pages:
+
+- Checkbox
+- Radio Group
+- Switch
```

Comment:
We need to make these hyperlinks, don't we?

Reply:
Fixed with https://github.com/wso2/oxygen-ui/pull/156/commits/0f9fa5b6e2dd61aa147c7aa60e68bfd1c97ec6be